mail_operating_unit/wizard/mail_compose_message_view_ext.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="email_compose_message_wizard_form_mail">
+        <field name="name">mail.compose.message.form.mail</field>
+        <field name="model">mail.compose.message</field>
+        <field name="inherit_id" ref="mail.email_compose_message_wizard_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='active_domain']" position="after">
+                <field name="operating_unit_id" invisible="1"/>
+            </xpath>
+            <field name="template_id" position="attributes">
```

Comment:
Should this be done by Access Rules? Have a look at product_operating_unit or others. 
The idea is that each operating unit just see what it is assigned on. To be sure about that, it's better to do it with access rules than limiting visibility on each view.

Reply:
strange, I did it like in operating_unit. there it is all done like this...